govc/vm/disk/change.go

```diff
@@ -152,6 +156,13 @@ func (cmd *change) Run(ctx context.Context, f *flag.FlagSet) error {
 		editdisk.CapacityInKB = int64(cmd.bytes) / 1024
 	}
 
+	current := editdisk.StorageIOAllocation.Limit
+	desired := cmd.limit
+
+	if desired != nil && *current != *desired {
```

Comment:
What if the value is not provided? Wouldn't `cmd.limit` be a `nil` pointer and panic on `*current` deref?

Reply:
I had my doubts about your initial response, then thought about it and you're right: I don't need a nil check because it won't be sent to the remote in this case. The new code works and requires no nil check :)